gooddata-java-model/src/main/java/com/gooddata/sdk/model/executeafm/afm/AttributeFilterElements.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2007-2019, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.sdk.model.executeafm.afm;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static com.fasterxml.jackson.databind.JsonMappingException.from;
+
+/**
+ * Represents elements, which can be used as value of "in" or "notIn" by {@link AttributeFilter}.
+ */
+@JsonSerialize(using = AttributeFilterElements.Serializer.class)
+@JsonDeserialize(using = AttributeFilterElements.Deserializer.class)
+public interface AttributeFilterElements {
+
+    List<String> getElements();
```

Comment:
Would be nice to have some javadoc here.

Reply:
javadoc written